### PR TITLE
PASS1-555: Use const where possible.

### PR DESCRIPTION
**Pull request by jeandudey**
```
Before:
   text    data     bss     dec     hex filename
1582224   19776  510336 2112336  203b50 build-Passport/firmware.elf

After:
   text    data     bss     dec     hex filename
1581600   19776  509336 2110712  2034f8 build-Passport/firmware.elf
```

Reduces `text` by 624 bytes and `bss` by 1000 bytes.

The `_` prefix on constants is to make MicroPython make that variable only available on that module.

Before merging I would like to test first:

<details>
<summary>check list</summary>

- [ ] check QR code view works.
- [ ] check utxo history works. (not sure how to test).
- [ ] check keypad works.
- [ ] check animations work.
- [ ] check FCC test works.
- [ ] check login flow works.
- [ ] check verify address flow works.
- [ ] check seed words list page works.
- [ ] make sure MainScreen works (if UI works should be ok).
- [ ] check font sizes are ok.
- [ ] check that file sha256 calculation works.
- [ ] check backup code task works.
- [ ] check ur1 works.
- [ ] check ur2 works.
- [ ] check backup code section works.
- [ ] check that battery indicator works.
- [ ] check that color picker works? is it used?
- [ ] check icon button works.
- [ ] check pin input works.
- [ ] check text input works.
</details>

### CODE REVIEW

File: ports/stm32/boards/Passport/modules/ur2/cbor_lite.py [hunk 1]
```diff
@@ -11,55 +11,57 @@
...
+Tag_Minor_convertBase16 = const(23)
+Tag_Minor_cborEncodedData = const(24)
+Tag_Minor_uri = const(32)
+Tag_Minor_base64Url = const(33)
+Tag_Minor_base64 = const(34)
+Tag_Minor_regex = const(35)
+Tag_Minor_mimeMessage = const(36)
+Tag_Minor_embeddedJSON = const(262)
+Tag_Minor_selfDescribeCbor = const(55799)
 Tag_Minor_mask = 0x1f
```
Comment (jeandudey) [inline ports/stm32/boards/Passport/modules/ur2/cbor_lite.py:65]:
```suggestion
Tag_Minor_mask = const(0x1f)
```

File: ports/stm32/boards/Passport/modules/screens/main_screen.py [hunk 2]
```diff
@@ -5,18 +5,14 @@
 #                  the StatusBar and CardNav components.
 
 import lvgl as lv
+from micropython import const
```
Comment (mjg-foundation) [inline ports/stm32/boards/Passport/modules/screens/main_screen.py:8]: Is const still needed here?

File: simulator/sim_modules/passport/sram4.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: 2021 Foundation Devices, Inc. <hello@foundationdevices.com>
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-MAX_QR_VERSION = 24
+MAX_QR_VERSION = 13
```
Comment (mjg-foundation) [inline simulator/sim_modules/passport/sram4.py:4]: Why was this changed, and why was it defined here and in modpassport-sram4.h?

File: ports/stm32/boards/Passport/modules/flows/fcc_test_flow.py [hunk 4]
```diff
@@ -16,9 +16,8 @@
 from uasyncio import sleep_ms
 from errors import Error
 
-CAMERA_DISPLAY_DURATION_SECS = 5 * 1000
-FILE_COPY_REPEAT_COUNT = 10
-FILE_SIZE_TO_CREATE = 250 * 1024
+_CAMERA_DISPLAY_DURATION_SECS = const(5 * 1000)
```
Comment (FoundationKen) [inline ports/stm32/boards/Passport/modules/flows/fcc_test_flow.py:19]: Not a fan of the leading underscore for "internal" constants.  I know Python people like it, but it's ugly, harder to read, and more work to type.

Comment (jeandudey) [reply ports/stm32/boards/Passport/modules/flows/fcc_test_flow.py:19]: Yeah, it's ugly I agree but sadly that's the way to save some bytes per global constant

**Comment by mjg-foundation** [comment]
Anything blocking this PR @jeandudey ?

**Review by FoundationKen — DISMISSED** [review]

File: simulator/sim_modules/passport/sram4.py (hunk 3 above)
Comment (mjg-foundation) [reply simulator/sim_modules/passport/sram4.py:4]: This seems to be defined multiple places, is it possible to consolidate to 1 global constant?

File: ports/stm32/boards/Passport/modules/screens/main_screen.py (hunk 2 above)
Comment (jeandudey) [reply ports/stm32/boards/Passport/modules/screens/main_screen.py:8]: Not needed, removing it

File: simulator/sim_modules/passport/sram4.py (hunk 3 above)
Comment (jeandudey) [reply simulator/sim_modules/passport/sram4.py:4]: I changed it to match the code in `modpassport-sram4.h`, we can't consolidate it yet because `sram4.py` simulates that module from `modpassport-sram4.h` so it's needed. What I really like to do in the future is have a `modpassport-sram4.h` defined for both unix and stm32 ports and using the preprocessor to choose between the native implementation and the hardware one.

**Comment by jeandudey** [comment]
@mjg-foundation updated the PR with a new commit, nothing blocking it

File: simulator/sim_modules/passport/sram4.py (hunk 3 above)
Comment (mjg-foundation) [reply simulator/sim_modules/passport/sram4.py:4]: It's also defined in modules/views/qrcode.py, could that be consolidated?

**Review by FoundationKen — APPROVED** [review]
If possible, we can consolidate the constants Matt mentioned in a later PR.
